package.json

```diff
@@ -78,5 +78,8 @@
       "style": "module",
       "parser": "typescript"
     }
+  },
+  "dependencies": {
+    "react-icons": "^4.6.0"
```

Comment:
react-icons is not used in this PR. Please revert changes to package.json and pnpm-lock.yaml as this was the only change made to both.